src/commands/synthetics/upload-application-command.ts

```diff
@@ -35,6 +36,41 @@ export class UploadApplicationCommand extends Command {
   }, LogLevel.INFO)
 
   public async execute() {
+    try {
+      await this.resolveConfig()
+    } catch (error) {
+      this.logger.error(`Error: invalid config`)
+
+      return 1
+    }
+
+    try {
+      const version = await uploadMobileApplicationVersion(this.config)
+      this.logger.info(`Created new version ${version.version_name}, with version ID: ${version.id}`)
+    } catch (error) {
+      if (error instanceof CiError || error instanceof EndpointError || error instanceof CriticalError) {
+        this.logger.error(`Error: ${error.message}`)
+      } else {
+        this.logger.error(`Error:F`)
```

Comment:
I am not sure to understand why we got this message.

Reply:
oops 🫠 - fixed!